persistence/scaladsl/src/main/scala/com/lightbend/lagom/internal/scaladsl/persistence/ReadSideImpl.scala

```diff
@@ -59,7 +59,7 @@ private[lagom] class ReadSideImpl(
     val streamName     = tags.head.eventType.getName
     val projectionName = readSideName
 
-    val readSideProps = (projectionRegistryActorRef: ActorRef) =>
+    val readSidePropsFactory = (projectionRegistryActorRef: ActorRef) =>
       ReadSideActor.props(
         streamName,
         projectionName,
```

Comment:
I still have doubts about `streamName` and `projectionName`. What's the difference and for what is that useful?

Reply:
I share the doubts. `streamName` will probably disappear.